apps/home/blocks.py

```diff
@@ -80,9 +81,10 @@ class Meta:
         icon = 'image'
 
 
+# RichTextField required to ensure same richtext options throughout site
+# editing fetaures of RichTextBlock broken in Wagtail4.1
 class ParagraphBlock(blocks.StructBlock):
-    body = blocks.RichTextBlock(
-        required=True,
+    body = fields.RichTextField(
```

Comment:
found that the RichTextField does use the default config but does not work when added this way